include/insns/RV64I.h

```diff
@@ -85,18 +85,23 @@ namespace SST{
 
```

Comment:
Changes from this PR that addressed bugs in RV64I instructions have been pushed to devel in commit b037382. Changes that just re-structured code are not included in that commit